dashboard/app/views/shared/_header_progress.html.haml

```diff
@@ -1,7 +1,8 @@
 - small_text = @script && (@script.name == "starwars" || @script.name == "starwarsblocks")
 .header_level
   %div.header_level_container
-    = link_to @stage.localized_title, @script, class: ["header_text", ('small_font_on_tablet' if small_text)]
+    - text = @script.professional_learning_course? ? @stage.localized_name : @stage.localized_title
+    = link_to text, @script, class: ["header_text", ('small_font_on_tablet' if small_text)]
```

Comment:
Going to leave this as-is for now since this is existing code (just moved to a new line).